src/types.ts

```diff
@@ -43,6 +43,9 @@ export type LawId = string
 export type LawLabel = "hidden" | "initial" | "Kernenergie"
 
 export type LawDefinition = {
+  __isFragment?: never
+  __isTeleport?: never
+  __isSuspense?: never
```

Comment:
What does that mean?

Reply:
Oops. This was a try, copy-pasting from vue. Didn't do any good. Will remove.